src/openms/source/ANALYSIS/QUANTITATION/TMTTenPlexQuantitationMethod.cpp

```diff
@@ -90,27 +102,23 @@ namespace OpenMS
     defaults_.setValue("reference_channel", "126", "The reference channel (126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131).");
     defaults_.setValidStrings("reference_channel", TMTTenPlexQuantitationMethod::channel_names_);
 
-    //    {0.0, 1.0, 5.9, 0.2},   //114
-    //    {0.0, 2.0, 5.6, 0.1},
-    //    {0.0, 3.0, 4.5, 0.1},
-    //    {0.1, 4.0, 3.5, 0.1}    //117
     defaults_.setValue("correction_matrix", ListUtils::create<String>("0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0"),
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0"),
                        "Correction matrix for isotope distributions (see documentation); use the following format: <-2Da>/<-1Da>/<+1Da>/<+2Da>; e.g. '0/0.3/4/0', '0.1/0.3/3/0.2'");
```

Comment:
this is the scheme Thermo uses in their certificates, so it should be clear to the user what to fill in.
But I agree, it could be more explcitily stated in the docs.
Maybe a doc-fix hour during the retreat would be good...